setup.py

```diff
@@ -112,14 +112,16 @@ def get_setuptools_script_dir():
       url='https://github.com/jobovy/apogee',
       package_dir = {'apogee/': ''},
       packages=['apogee','apogee/tools','apogee/select','apogee/test',
-                'apogee/util','apogee/samples'],
+                'apogee/util','apogee/samples','apogee/spec','apogee/modelatm',
+                'apogee/modelspec'],
       package_data={'apogee/samples':['data/rcmodel_mode_jkz_ks_parsec_newlogg.sav',
                                       'data/rcmodel_mode_jkz_h_parsec_newlogg.sav',
                                       'data/rcmodel_mass_agez.sav',
                                       'data/rcmodel_mass_agez_coarseage.sav',
                                       'data/rcmodel_omega_agez.sav'],
                     'apogee/spec':['filter/dr12/*.filt',
-                                   'cannon/training/*.txt'],
+                                   'cannon/training/*.txt',
+                                   'cannon/trained/*.txt'],
```

Comment:
Good catch! I always install my code with

``python setup.py develop``

so I don't notice when things don't get properly copied to the installation directory!